ocs_ci/ocs/ui/views.py

```diff
@@ -1157,6 +1157,19 @@
 
 
 locators = {
+    "4.13": {
+        "login": {**login, **login_4_11},
+        "page": {**page_nav, **page_nav_4_10},
+        "generic": generic_locators,
+        "deployment": {
+            **deployment,
+            **deployment_4_7,
+            **deployment_4_9,
+            **deployment_4_10,
+            **deployment_4_11,
+            **deployment_4_12,
+        },
+    },
```

Comment:
Same as above, I'm not sure, if we should keep the `4.13` related changes in the backport for `release-4.12` branch.

Reply:
done